tests/testthat/test-methods-epi_df.R

```diff
@@ -0,0 +1,21 @@
+test_that("head and tail do not drop the epi_df class", {
+  tib <- tibble::tibble(
```

Comment:
`tib` sounds like it should be a tibble, but this is an `epi_df`; maybe rename to something else.

Reply:
Done.